src/main/java/io/vinyldns/java/serializers/InstantSerializer.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2018 Comcast Cable Communications Management, LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.vinyldns.java.serializers;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+import java.lang.reflect.Type;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+public class InstantSerializer implements JsonSerializer<Instant>, JsonDeserializer<Instant> {
```

Comment:
Doesn't Google gson support Instant out of the box?

Reply:
Unfortunately not for serialization; GSON serializes `java.time.Instant` as the data structure (I believe `nanos ` and `epochSeconds`). We also had to write a custom serializer for `org.joda.time.DateTime` as well.

Reply:
boooooooooo